pkg/v1/providers/config_default.yaml

```diff
@@ -605,6 +605,7 @@ ANTREA_TRACEFLOW: true
 ANTREA_EGRESS: false
 ANTREA_FLOWEXPORTER: false
 ANTREA_DISABLE_UDP_TUNNEL_OFFLOAD: false
+ANTREA_DEFAULT_MTU: null
```

Comment:
Question: is this variable intended to be provided by user explicitly via `export ANTREA_DEFAULT_MTU=xxx`?

Reply:
BTW, this var should be a string according to the schema file defined. https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/antrea/1.2.3/bundle/config/schema.yaml#L24

Reply:
Thanks @blc1996 for your question, @12345lcr could you provide more context here? How does user use this "DEFAULT_MTU" variable? Do you have a t-f pr for calico that I can take a look? Thanks!